README.rst

```diff
@@ -101,6 +101,20 @@ as long as you see ``Successfully installed`` at the end.
 .. _issue #61:
    https://github.com/IntelPNI/brainiak/issues/61
 
+
+Examples
+========
+
+We include BrainIAK usage examples in the examples directory of the code
```

Comment:
I think the name shown there is self explanatory: "srm_image_prediction_example". And we do have notebooks for other examples as well.